extensions/provisioning/customcluster.go

```diff
@@ -1,9 +1,19 @@
 package provisioning
 
 import (
+	"github.com/rancher/shepherd/pkg/config"
+	"github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
 )
 
+const (
+	installDockerConfigFileKey = "installDocker"
+)
+
+type InstallDockerConfig struct {
```

Comment:
@slickwarren @caliskanugur should we denote that the InstallDockerConfig is only for rke1 custom clusters?

Reply:
Last thing and then please create imports. Should this be a part of the provisioningInputs and name it `RKE1CustomClusterDockerInstall`?  @slickwarren

Reply:
yes I think that makes more sense

Reply:
Added it in provisioningInputs and named it RKE1CustomClusterDockerInstall